contrib/Zipkin/SpanConverter.php

```diff
@@ -43,7 +42,6 @@ private function sanitiseTagValue($value)
 
     public function convert(Span $span)
     {
-        $timestamp = Clock::get()->timestamp();
         $start_realtime = $span->getStartTimestamp();
```

Comment:
I'm thinking we should probably use the monotonic clock instead of the realtime clock for this one:

https://github.com/open-telemetry/opentelemetry-php/blob/master/sdk/Trace/Clock.php#L40-L44

This is an artifact left from https://github.com/open-telemetry/opentelemetry-php/issues/73

@mattiabasone - are you interested in taking on that work alongside this PR?

Reply:
@bobstrecansky so these changes don't fix the problem?

Before this "fix" I have some issues with trace representation because root span (last closed) had the greatest timestamp value and jaeger tries to adjust clock skew

Before
![tracing_before](https://user-images.githubusercontent.com/3951536/87938408-d8d58b00-ca96-11ea-884a-573ada731f20.png)

After
![tracing_after](https://user-images.githubusercontent.com/3951536/87938414-da9f4e80-ca96-11ea-9b8f-c843c220149c.png)

Reply:
@mattiabasone - I think your fixes do fix the imminent problem.  I think we need to use a monotonic clock in a couple places in these calls as well, as that'll be a more accurate time call.

Reply:
@bobstrecansky probably missing something, but if all the monotonic clocks are sent to API, isn't there the risk of having many different timestamps (based on server/container system clocks) if you work with different microservices?

Surely it can be used to measure the total duration of the span, saving in two properties startTime/endTime the value of monotonic clock at two different times.

Reply:
Agree with you that there is a risk of clock skew based on different services. Also agree that the monotonic clock should be used to measure the duration of the span in the way perscribed.

https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#time